frontend/src/registration/components/RegistrationNavButtons/styles.scss

```diff
@@ -2,7 +2,13 @@
   display: inline-block;
   width: 100%;
   padding-top: 32px;
-
+  
+  .float-left {
+    font-size: 14px;
+  }
+  .float-right{
+    font-size: 14px;
+  }
```

Comment:
@xitij2000 Sorry, I have zilch experience with CSS: My understanding is that this styles all objects with class name `float-left`/`float-right` within elements of class `registration-nav` will be styled. 
As I want their font size to be consistent across pages, is this not the way to go about it? isn't `float-left/right` just a name like `foo` or `bar` here? :thinking:

Reply:
It technically will work, but isn't a good style for a couple of reasons:

- The class names  `float-left` and `float-right` have nothing to do with font size, so it is unexpected that applying those classes will change the font size of an element. 
- It potentially introduces subtle bugs. Any future element in this UI that uses those classes will automatically get a font size change, and while that's obvious now, future developers might find it hard to debug since it's unexpected.
- float-left and float-right are like foo and bar only if you feel that a variable name has no relevance to its contents. It is bad style to name meaningful variables as a, b, c, z, y, z, i etc, and similarly we wound never name a css class foo or bar. 
- Names like float-left etc represent utility classes and as such they should behave accordingly. In contrast, a class name like "nav-button" is more free, a nav button could imply any font size, any font colour etc based on the theming of the site.

Reply:
Additionally, try to match the code style in the rest of the file by:
- adding a space after the class name
- removing the blank line before the class name.

Reply:
>float-left and float-right are like foo and bar only if you feel that a variable name has no relevance to its contents. It is bad style to name meaningful variables as a, b, c, z, y, z, i etc, and similarly we wound never name a css class foo or bar.

That is not the point I was trying to make: `foo` and `bar` are examplatory names for any kind of variable or object: They have no special meaning to the language. And if I understood you correctly, that's also the case for `float-left/right`. 

Thanks for clarifying that!